src/Maestro/SubscriptionActorService/.config/settings.json

```diff
@@ -6,6 +6,7 @@
     "GitHubAppId": "[vault(github-application-id)]",
     "PrivateKey": "[vault(github-application-private-key)]"
   },
+  "GitDownloadLocation": "https://netcorenativeassets.blob.core.windows.net/resource-packages/external/windows/git/Git-2.22.0-64-bit.zip",
```

Comment:
>https://netcorenativeassets.blob.core.windows.net/resource-packages/external/windows/git/Git-2.22.0-64-bit.zip [](start = 26, length = 110)

I guess if you're developing on a linux machine you already have git, so it should be fine to have only the windows version for the cluster.

Reply:
Yeah, I'm not super concerned with Linux versions of the cluster at the moment.  This could certainly be overriddden in your Development.json file.

Reply:
Yeah Agreed